examples/petsc/multigrid.c

```diff
@@ -148,6 +173,44 @@ int main(int argc, char **argv) {
       dmorig = dmDist;
     }
   }
+  {
```

Comment:
Should this be activated via a command line option?

Reply:
The transform with `eps = 1` is the identity so it seems not worth deciding whether to apply. The default makes no transforamtion.

Reply:
good point